conf/alluxio-site.properties.template

```diff
@@ -13,71 +13,17 @@
 # Details about all configuration properties http://www.alluxio.org/documentation/en/Configuration-Settings.html
 
 # Common properties
-# alluxio.debug=false
-# alluxio.home=/mnt/alluxio_default_home
-# alluxio.test.mode=false
 # alluxio.underfs.address=${alluxio.work.dir}/underFSStorage
-# alluxio.web.threads=1
-
-# Integration properties
-# alluxio.integration.master.resource.cpu=1
-# alluxio.integration.master.resource.mem=1024MB
-# alluxio.integration.worker.resource.cpu=1
-# alluxio.integration.worker.resource.mem=1024MB
 
 # Security properties
 # alluxio.security.authentication.type=SIMPLE
-# alluxio.security.authorization.permission.enabled=true
-# alluxio.security.authorization.permission.supergroup=supergroup
-# alluxio.security.group.mapping.class=alluxio.security.group.provider.ShellBasedUnixGroupsMapping
-
-# Master properties
-# alluxio.master.bind.host=0.0.0.0
-# alluxio.master.port=19998
-# alluxio.master.retry=29
-# alluxio.master.web.bind.host=0.0.0.0
-# alluxio.master.web.port=19999
-# alluxio.master.whitelist=/
-# alluxio.master.worker.threads.max=2048
-# alluxio.master.worker.threads.min=512
-# alluxio.master.worker.timeout.ms=300000
-# alluxio.master.tieredstore.global.levels=3
-# alluxio.master.tieredstore.global.level0.alias=MEM
-# alluxio.master.tieredstore.global.level1.alias=SSD
-# alluxio.master.tieredstore.global.level2.alias=HDD
 
 # Worker properties
-# alluxio.worker.bind.host=0.0.0.0
-# alluxio.worker.block.threads.max=2048
-# alluxio.worker.block.threads.min=256
-# alluxio.worker.data.bind.host=0.0.0.0
-# alluxio.worker.data.folder=/alluxioworker/
-# alluxio.worker.data.port=29999
-# alluxio.worker.file.persist.pool.size=64
 # alluxio.worker.memory.size=1GB
-# alluxio.worker.port=29998
-# alluxio.worker.session.timeout.ms=10000
 # alluxio.worker.tieredstore.levels=1
 # alluxio.worker.tieredstore.level0.alias=MEM
 # alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk
-# alluxio.worker.tieredstore.level0.dirs.quota=${alluxio.worker.memory.size}
-# alluxio.worker.tieredstore.level0.reserved.ratio=0.1
-# alluxio.worker.web.bind.host=0.0.0.0
-# alluxio.worker.web.port=30000
 
 # User properties
-# alluxio.user.block.master.client.threads=10
-# alluxio.user.block.worker.client.threads=128
-# alluxio.user.block.remote.read.buffer.size.bytes=8MB
-# alluxio.user.block.size.bytes.default=512MB
-# alluxio.user.failed.space.request.limits=3
-# alluxio.user.file.buffer.bytes=1MB
-# alluxio.user.file.cache.partially.read.block=true
-# alluxio.user.file.master.client.threads=10
 # alluxio.user.file.readtype.default=CACHE_PROMOTE
-# alluxio.user.file.seek.buffer.size.bytes=128MB
-# alluxio.user.file.write.location.policy.class=alluxio.client.file.policy.LocalFirstPolicy
-# alluxio.user.file.writetype.default=MUST_CACHE
-# alluxio.user.lineage.enabled=false
-# alluxio.user.lineage.master.client.threads=10
-# alluxio.user.ufs.delegation.enabled=false
```

Comment:
Okay, I thought it might worth leaving the delegation flag here as an example? because `1.3` is the first release that we enable delegation by default. Thus would make users a bit easier to identify this particular flag in case someone would like to keep this as the old value `false`. But I don't have a strong preference.